aqua/qt53/Portfile

```diff
@@ -0,0 +1,1775 @@
+dist_subdir qt5
+
+# file sizes are significantly smaller using xz
+if { ${subport} ne ${name} && ${subport} ne "${name}-docs" } {
+    use_xz yes
+}
+
+conflicts qt3 qt3-mac
+# conflict with all other version of qt5
+foreach {qt_test_name qt_test_info} [array get available_qt_versions] {
+    if {${name} ne ${qt_test_name}} {
+        conflicts-append [lindex ${qt_test_info} 0]
+    }
+}
+
+############################################################################### Modules Not Considered
+#
+# No qtactiveqt      (Windows Only)
+# No qtandroidextras (Android Only)
+# No qtwayland       (Linux Only)
+# No qtx11extras     (X11 Only)
+# No qtwinextras     (Windows Only)
+#
+###############################################################################
+
+############################################################################### Finding Dependencies
+#
+# grep -r "%dependencies" *
+# find ./ -name sync.profile
+#
+# grep -r qtCompileTest *
+#
+# find ./ -name config.tests
+#
+# find ./ -name Find\*.cmake
+#
+# grep -r qtHaveModule *
+#
+# grep -r packagesExist *
+#
+# port provides `find ./ -name \*.dylib -exec otool -L {} \; | grep /opt/local/libexec/qt5 | cut -d ' ' -f1` | cut -d : -f2 | sort -u
+# port provides `find ./ -name \*.dylib -exec otool -L {} \; | grep /opt/local/lib/ | cut -d ' ' -f1` | cut -d : -f2 | sort -u
+#
+# https://code.qt.io/cgit/qt/qt5.git/tree/.gitmodules?h=5.5.1
+#
+###############################################################################
+
+############################################################################### Notes
+#
+# qtconnectivity checks for bluetooth libraries
+#    bluez and bluez_le (Linux Only)
+#
+# qtconnectivity depends on
+#    qtandroidextras    (Android Only)
+#
+# qtserialport depends on
+#    ntddmodm           (Windows Only)
+#
+# qttools depends on
+#    qtactiveqt         (Windows Only)
+#
+# qtwebkit has tests for libraries in:
+#     Tools/qmake/mkspecs/features/configure.prf
+#     Tools/qmake/config.tests/
+#
+# the following qt modules found in qt5.5 don't exist in qt5.3 and were removed
+# qt3d qtcanvas3d qtwebengine qtwebchannel
+#
+###############################################################################
+
+############################################################################### TODO
+#
+# TODO: possible to trim dependencies of qtbase?
+#
+###############################################################################
+
+############################################################################### Module Format
+#
+# "Qt Module Name" {
+#     {
+#         checksum, rmd160
+#         checksum, sha256
+#         checksum, size
+#     }
+#     dependencies, build
+#     dependencies, lib
+#     dependencies, Qt module name
+#     Qt components provided
+#     included in "standard" installation of Qt (empty string is no, explanation string is yes)
+#     variant overrides
+#     revision number
+#     license replacement
+# }
+#
```

Comment:
For consistency with other Qt 5 ports, perhaps we could add
`# module info found at http://web.archive.org/web/20160310083902/http://doc.qt.io/archives/qt-5.3/index.html`